src/assets/thirdPartyServices/snet/named_entity_recognition/index.js

```diff
@@ -1,81 +1,88 @@
+import parse from 'html-react-parser';
 
 export default class NamedEntityRecognitionService extends React.Component {
   constructor(props) {
     super(props);
     this.submitAction = this.submitAction.bind(this);
     this.handleFormUpdate = this.handleFormUpdate.bind(this);
-    this.handleChange = this.handleChange.bind(this);
-    this.canBeInvoked = this.canBeInvoked.bind(this);
+    this.handleParameterChange = this.handleParameterChange.bind(this);
+    this.handleInputText = this.handleInputText.bind(this);
+
+    this.textInput = React.createRef();
 
     this.state = {
-      value: "",
-      serviceName: "RecognizeMessage",
-      methodName: "Recognize",
-      message: undefined,
-      styles: {
-        details: {
-          fontSize: 14,
-          alignItems: "center",
+      selectedExample: 0,
+      exampleList: [
+        {
+          label: "Custom text",
+          value: 0,
+          content: "",
         },
-        defaultFontSize: {
-          fontSize: 15,
+        {
+          label: "Example 1",
+          value: 1,
+          content: "Our concept of operations is to flow in our military assets with a priority to build up southern " +
+            "Texas, and then Arizona, and then California, Donald Trump said Monday, adding that the soldiers normally " +
+            "assigned weapons will be carrying them at the border.  We'll reinforce along priority points of entry, and " +
+            "while this happens, Trump Hotels is falling down in stock market.",
         },
-      },
+      ],
+      serviceName: "RecognizeMessage",
+      methodName: "Recognize",
+      message: "",
+      response: undefined,
+      reqStart: undefined,
     };
-
-    this.serviceMethods = [];
-    this.allServices = [];
-    this.methodsForAllServices = [];
   }
 
   handleFormUpdate(event) {
-    console.log(event.target);
     this.setState({ [event.target.name]: event.target.value });
-  }
+  };
 
-  submitAction() {
-      const { methodName } = this.state;
-      const methodDescriptor = RecognizeMessage[methodName];
-      const request = new methodDescriptor.requestType();
-      request.setValue(JSON.stringify(this.handleSentences()));
-      const props = {
-        request,
-        onEnd: ({ message }) => {
-          this.setState({value: message.getValue()});
-        },
-      };
-      this.props.serviceClient.unary(methodDescriptor, props);   
+  handleParameterChange(event) {
+    this.setState({
+      selectedExample: event.target.value,
+    }, () => {
+      if (this.state.selectedExample !== "default") {
+        this.state.exampleList.forEach(item => {
+          if (item.value == event.target.value) {
+            this.setState({ message: item.content }, () => {
+                this.textInput.current.inputRef.current.focus();
+              },
+            );
+          }
+        });
+      }
+    });
   }
 
-  handleChange(event) {
+  handleInputText(event) {
+    console.log(event.target);
```

Comment:
please remove console.logs, this will expose unnecessary info to the user console.

Reply:
yep.. done.

Reply:
thank you